src/components/utils/peoplePicker.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ArrowSVG from '../svg/ArrowSVG.jsx';
+
+class PeoplePicker extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      count: 0,
+    };
+    this.increase = this.increase.bind(this);
+    this.decrease = this.decrease.bind(this);
+  }
+
+  increase(e) {
+    e.preventDefault();
+    let currCount = this.state.count;
+    if (currCount !== 50) {
+      currCount += 1;
+    }
+    this.setState({count: currCount});
+  }
+  decrease(e) {
+    e.preventDefault();
+    let currCount = this.state.count;
+    if (currCount !== 0) {
+      currCount -= 1;
+    }
+    this.setState({count: currCount});
```

Comment:
Czemu nie tak ?
if (this.state.count !== 0) {
      this.setState({count: this.state.count - 1 });
   }